build/jre/src/Downloader.java

```diff
@@ -1,3 +1,26 @@
+/* -*- mode: java; c-basic-offset: 2; indent-tabs-mode: nil -*- */
+
+/*
+  Part of the Processing project - http://processing.org
+
+  Copyright (c) 2012-19 The Processing Foundation
+  Copyright (c) 2004-12 Ben Fry and Casey Reas
```

Comment:
This file has only existed since 2014, so the only copyright is the Foundation, from 2014-2019.

Reply:
Got it. Updating now!